guide/guide/.js/src/main/scala/io/udash/web/guide/views/ext/demo/bootstrap/NavigationDemo.scala

```diff
@@ -36,11 +39,24 @@ object NavigationDemo extends AutoDemo {
           case MenuContainer(name, children) =>
             val dropdown = UdashDropdown(children.toSeqProperty)(
               (item, _) => linkFactory(item.get),
-              _ => span(name, " ")
+              _ => span(name, " "),
+              buttonFactory = UdashButton(
```

Comment:
I like the code change, but now all those demos became more complex - is it an issue with the API or the snippets?

Reply:
I guess that it was necessary to make demos slightly more complicated